playbook.md

```diff
@@ -30,7 +30,7 @@ We think that it's very important to have a talented team if we're going to succ
 We are helpful, useful and expert. We give practical and pragmatic advice to each other, and to our clients.
 
 #### Honest
-We are honest, trustworthy and straightforward. We give plain-spoken, frank, accurate feedback and advice, and we never mislead or obfuscate.
+We are honest, trustworthy and straightforward. We give plain-spoken, frank, accurate feedback and advice, and we never mislead or wilfully confuse.
```

Comment:
That's not how I understand the meaning of that word, but regardless, I feel like we should take our steer from what the _intent_ of this paragraph is, rather than what the previously used word means. 

For me, the point is that we don't tell clients that things are OK when they're not, and we don't hide things from them